pkg/kv/local/driver.go

```diff
@@ -0,0 +1,76 @@
+package local
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/dgraph-io/badger/v3"
+	"github.com/treeverse/lakefs/pkg/kv"
+	kvparams "github.com/treeverse/lakefs/pkg/kv/params"
+	"github.com/treeverse/lakefs/pkg/logging"
+)
+
+const (
+	DriverName           = "local"
+	DefaultDirectoryPath = "~/data/lakefs/kv"
+	DefaultPrefetchSize  = 256
+)
+
+var (
+	driverLock    = &sync.Mutex{}
+	connectionMap = make(map[string]*Store)
+)
+
+type Driver struct{}
+
+func normalizeDBParams(p *kvparams.Local) {
+	if len(p.DirectoryPath) == 0 {
+		p.DirectoryPath = DefaultDirectoryPath
+	}
+	if p.PrefetchSize == 0 {
```

Comment:
```suggestion
	if p.PrefetchSize <= 0 {
```
To be on the safe side